sources/packages/forms/src/form-definitions/programinformationrequest.json

```diff
@@ -2353,12 +2353,12 @@
           "tags": [],
           "properties": {},
           "conditional": {
-            "show": null,
+            "show": "",
             "when": null,
             "eq": "",
             "json": ""
           },
-          "customConditional": "",
+          "customConditional": "show = data['isReadOnlyUser'] !== true;",
```

Comment:
There is no need to string property access, it can be `show = data.isReadOnlyUser !== true`